packages/web/src/types/sanity.ts

```diff
@@ -123,6 +123,11 @@ export interface ApproachPage extends DocumentBase {
   >
 }
 
+export interface PrivacyPage extends DocumentBase {
+  meta?: Meta
+  card?: Card
```

Comment:
we actually don't need to add `meta` and `card` because they're included in `DocumentBase` maybe worth removing and subsequently change the other types where we've done that.